app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
Huh. I didn't see currentPlan in my store when I went to look for it after switching from the Plan component to the Treatment component. (It was null.) Maybe I'm doing something wrong?